relaybot.py

```diff
@@ -52,7 +53,15 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        optionAsBoolean = { "": False, "false": False, "no": False, "true": True, "yes": True }
+        sentinel = object()
+        ssl = options.get('ssl', sentinel)
+        if(sentinel == ssl):

```

Comment:
Does it work better to go with explicit membership checks?

    ssl = options['ssl']
    if ssl in optionAsBoolean:
        if optionAsBoolean[ssl]:
            reactor.connectSSL(options['host'], int(options['port']), factory, ClientContextFactory(), int(options['timeout']))
        else:
            reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
    else:
        raise TypeError("Cannot convert '{}' to boolean.".format(ssl))